src/jest/config.json

```diff
@@ -13,7 +13,8 @@
     "!ui_framework/components/**/*/index.js"
   ],
   "moduleNameMapper": {
-    "^ui_framework/components": "<rootDir>/ui_framework/components"
+    "^ui_framework/components": "<rootDir>/ui_framework/components",
+    "\\.(css|less|scss)$": "<rootDir>/src/jest/mockCssFiles.js"
```

Comment:
Want to borrow something similar from https://github.com/elastic/kibana/pull/13172/files#diff-68998b8839f75d151a840687485895a5R16?

```
"\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/ui_framework/src/test/file_mock.js"
```

It encompasses more files, which will be useful once we introduce SVGs (and I guess MP3s?) 😄